src/extension/wps/wps-core/src/test/java/org/geoserver/wps/ExecuteTest.java

```diff
@@ -751,6 +755,43 @@ public void testFeatureCollectionFileReferenceKVPError() throws Exception {
                 "/wps:ExecuteResponse/wps:Status/wps:ProcessFailed/ows:ExceptionReport/ows:Exception/ows:ExceptionText",
                 d);
     }
+    
+    @Test
+    public void testHttpSecutiry() throws Exception {
+        URLEntry googleOnly =
+                new URLEntry(
+                        "google only",
+                        "only allow url starting with http://www.google.com",
+                        "^(http://www.google).*$");
+        addURLEntryGeoserverURLConfigService(googleOnly);
+        //GeoserverURLConfigService.getSingleton().addAndsave(googleOnly);
```

Comment:
Please do not leave commented out bits of code. Also, the formatter would have put a space between "//" and "GeoserverURLConfigService", so this did not go through a local maven build